modules/document_repository/templates/menu_document_repository.tpl

```diff
@@ -128,7 +128,7 @@
                             <div class="visible-xs col-xs-12"> </div>
                             <div class="visible-xs col-xs-12"> </div>
                             <div class="col-sm-2">
-                                <input type="button" name="reset" value="Clear Form" class="btn btn-sm btn-primary col-xs-12" onclick="location.href='main.php?test_name=document_repository&reset=true'" />
+                                <input type="button" name="reset" value="Clear Form" class="btn btn-sm btn-primary col-xs-12" onclick="location.href='{$baseURL}/document_repository&reset=true'" />
```

Comment:
missing trailing slash after ModuleName.
And the first parameter needs to be preceded with a '?' instead of a '&'